### ISSUE TITLE
src 1::ff11 fails to parse, unlike with libpcap

### ISSUE THREAD

**Issue by kbara** [Igalia/pflua#248]
```src 1::ff11``` is valid pflang, and compiles with tcpdump:

```
tcpdump -i wlan0 -d "src 1::ff11"      
(000) ldh      [12]
(001) jeq      #0x86dd          jt 2	jf 11
(002) ld       [22]
(003) jeq      #0x10000         jt 4	jf 11
(004) ld       [26]
(005) jeq      #0x0             jt 6	jf 11
(006) ld       [30]
(007) jeq      #0x0             jt 8	jf 11
(008) ld       [34]
(009) jeq      #0xff11          jt 10	jf 11
(010) ret      #65535
(011) ret      #0
```

However, it does not work with pflua:

```
% ./pflua-expand 'src 1::ff11'
luajit: ../src/pf/parse.lua:309: 
Pflua parse error: In expression "src 1::ff11"
                                     ^
unknown src type 1 

stack traceback:
	[C]: in function 'primitive_error'
	../src/pf/parse.lua:309: in function 'error'
	../src/pf/parse.lua:474: in function 'parse_primitive_or_arithmetic'
	../src/pf/parse.lua:904: in function 'parse_logical_or_arithmetic'
	../src/pf/parse.lua:936: in function 'parse_logical'
	../src/pf/parse.lua:949: in function 'parse'
	./pflua-expand:31: in main chunk
	[C]: at 0x00405010

```

**Comment by kbara** [comment]
Note that ```host 1:ff11``` works with pflua-expand.

**Comment by mpeterv** [comment]
Looks like it's equivalent to `src host 1::ff11`, same for `dst`, dispatcher for these primitives needs a default to assume the second argument is host. I'll send a PR.

**Comment by kbara** [comment]
Thank you!

**Comment by kbara** [comment]
This is closed by #249 ; thanks!
